purchase_order_generator_revision/static/description/icon.svg

```diff
@@ -0,0 +1,79 @@
HrwvN3K6MNmCcOW8X6z xyLl7jbcx3SugP1mkaZp88ePH//+ww8/fADARQD0ZMcMEPnuCz/4wQ+mhoeHfz85Ofmn8Xh8xLIs 58ON7MwfsNTGekHErgYRn63DfhaG3ejBbft57YWx4Z3Atm3UajXUarU2Jjp//vxbDz/88CMHDx58 HcA8AKr+l+UIdONuCIAhAJMvvfTSP+3cufMfFEVJ0YdK9wgM2qxRxB8Iq/op6A6dADz3KxSRcvf5 IKn2O0fDvWq12vYWc7VaPX/gwIHnXnzxxWMAsohA6llEv/9KCzqAK2+//fav6/X6b3bu3LkzmUxu AOBs186u7acQlURRb9uLcTRNc865NYBb0kWZqpNyes6yLFQqlTapNwxDPXv27OEvf/nL//zhhx/+ N4ACIpL6tv6jasin/SSAW5988sm//eIXv/hQPB53dn8aHR3FTTfdxP0IQycawK8ui1Kp5Dzo8fHx JeeDJNZdvhw/QFVVqKraVrdQKJx67rnnnj18+PBbABZxjeCRzwx1SwOwMAFcee+99/7wu9/97sTn P//5balUajMA1Ot1VKtVZ3GIiAbwKg/yI9g6bg3gVZd3HJUPYJomCoVC2/5Cuq6rH3zwwQt33333 v54+ffo9AOrVU12bEuwFAwCtG6jOzMxcOnTo0C/GxsYuTE5O7pZlOWUYBkqlkrN8mr7bF0b1ixCF rcMuNmWXbPOI3ElYGoRqtYpCodDm4Wez2Z8+8MADj7788ssHAFxGF9Q9D71iAAoTwPyJEyf+cOrU qd/cfvvtqZGRkZ1Ayy6Xy+W2tQN+dln0wfOYxssHCINOCN9oNJDP59v6r9fr84cPH97/zW9+8+m5 ublfAqAqobsLAa6i1wwAtG6sPjs7e+Hw4cMfjY2NXdi8efP2eDw+YpomSqUSGo0GUqlUmzYIq4L9 HEU2DOx0e9owqVjLslAqlZytYylOnz596NFHH33yzTff/E8AOfRI6ll02wkM6psAGAGw6+DBg/+4 Y8eOe+hJWZaxceNGjIyMhCK6SHmxWHR+j46Ocq8RTd4EOXu1Wm3J/kKFQuHU66+//u9XQ7tprADh KVZCA7CwAWgAskePHv29JEmnt27duiWZTG6gUtNoNJYkkNwI6yxSHwAAdw8BnnTzsoN+MAwD+Xwe 5XK5LbT77W9/+9pdd9313Q8//PBtdNnDF8FKagAWdBxpAH/2wgsv3POZz3zmHkVRUkBrhdHExERb yCaaROKVUw1g2zbWr1+/pJ6XxPPKeBqgVCphcXGxTerz+fwvnn/++ZePHTv2X2ip+xUlPMVKawA3 mgAu/ehHPzq7uLj4f7t27dqeTCY32LaNSqWCarXqfLjJDb+IwF3O+gCswxmU+2fP8TQB3Tq2Uqm0 Sf2777779De+8Y1nz5w58y6AMvqE+ED/aAAWdEwpAH/+xBNP3LFnz569rDYYHx/HxMRE4D5AXmAT QZlMZsl5vzQwW07/LMty9hJk+z5//vxbTz755KGpqal30crk9Q3hKfpNA7DQAXzy/vvvT09NTX3w uc99bnR4eHgbzZ6Vy2Xu1zgAfxPgDgN514vOBgKtZNbs7KyzTTwAqKp6/siRI09/5zvfeWlubu5/ cC2hA/QR8YH+ZgCKxbm5uUtvvPHGhfXr1/9h27Ztu2RZTum6jsXFRZimiaGhoVAmgE0E8cyJiANo miZmZ2dx5cqVtoTO6dOnD33lK1/57okTJ95Ba8p2xTx8EawGBgBa2uDCz3/+8+ypU6dO3n777YmR kZEdQGuPwMXFRWfpVNAkEdBiAFru9V4eBc/5K5VKuHz5cls0USwWTz311FP/sn///kMAfo2W1Ped yndjtTAA0PINirOzs+cOHz48C+DXO3bs2EETSIVCAbquI5VKQZZlXzPAfjHciwF46p/uU8xOJ+u6 rp48efK1u++++9/OnDnzUwAziHjKtpvoRyfQD+x4JwHcdvDgwb+nCSRCWm8j3XLLLUsSSOxv1gn0 2kuQgtbL5/NOaEfbm5+f/8Wjjz763NTU1K/QSugYaBG97wlPsZo0AAsCoAjg0tGjR3O5XO5nu3fv 3pZIJDbYto1isQhVVT21AWsC/DaGAFom5vLly22hnaZp8z/84Q/3f+tb33plbm7uZwDyWEVSz2K1 aQA36Pg3A9j14osv3vXpT3/6HrrmIBaLYWJiAjfffHPbRWx2jre3MA3tqNQ7nRGC8+fPv7V///5D U1NT/wtgFi3CryqpZ7FaNQALglZy5ZN33nmncOXKlV/u3LlzQyqV2mzbrR1Dy+VyWwKJ1QC8xSiV SgXT09NQ1WvRm6qq51955ZV9jzzyyMGrUl/CKpV6FqtdA7Cg97IJwK4nnnjiL++4444HZFl2VnxM TExg06ZNKJfLAFqS7n6BNZfLoVKpOGWmaaqnTp06dP/9978J4JcAKlgF3r0o1hIDANfuRwHwhd27 d29+/PHH79+4ceNf0wqJRALDw8POHACdZCoUCsjn820vXxSLxVPPP//8c8eOHfsVgI/QiunXDPGB tWECeLAAZHO5XPmNN944NzIy8tH27dt3ybKcort/0IWplmUhm822zdWbpqn+5Cc/efree+/9/tX8 fc9W6PQaa00DsKD3lgDwF7t37775oYce+ht2zQEvVzAzM/PTffv2vTw1NfUbABdwzckD1hjxgbWr ASgIWgSczuVyhSNHjuRs2/7Vzp07/yQej48A15hA07T5Y8eO7f/2t7/9H3Nzc+8jwpcv+hlrWQOw oPcZB/AFAJnXXnvt77Zv3/41WZZTZ86cOfT1r3/9EFp2/hOsQVvvheuFAYBr9yoB2ADgs1/96lcn AODtt98+C+AkWgsyKfHXNOEpVv2eQSFACWqhlcBZIIQMNhoNCa08goXrQOW7cT1pABasNgDaJf66 IT5w/TIABXv/1xXhb+AGAAD/D0UpzXih8+WrAAAAAElFTkSuQmCC "
```

Comment:
You do not need to add images on bas64 anymore you can add them as links to your static folder in the module.